package.json

```diff
@@ -62,6 +62,9 @@
   "peerDependencies": {
     "ember-cli-babel": "^6.7.1"
```

Comment:
Shouldn't this be changed to accommodate babel@7?

Reply:
Yes, it should, I had intended to!